modules/k8s/mysql-ha-simple/outputs.tf

```diff
@@ -0,0 +1,3 @@
+output "application_name" {
+  value = juju_application.mysql.name
+}
```

Comment:
You could also expose `provides` and `requires` endpoints. See https://github.com/canonical/analytics-terraform-solutions/pull/2#discussion_r1785218066

Reply:
good call 
320f8b2